File: hw/tipan/u2/hw14MongoDb/SafeStoreJava/src/main/java/ec/edu/espe/safestore/model/MongoConnection.java
package ec.edu.espe.safestore.model;

import com.mongodb.client.MongoClient;
import com.mongodb.client.MongoClients;
import com.mongodb.client.MongoDatabase;

public class MongoConnection {

    private static final String URI =
        "mongodb+srv://Ronald:Ronald@cluster0.cd2ybxo.mongodb.net/?appName=Cluster0";

    private static MongoClient client;

    public static MongoDatabase getDatabase() {

        if (client == null) {
            client = MongoClients.create(URI);
        }

        return client.getDatabase("SafeStoreDB");
    }
}

File: hw/chavez/u3/hw17OPPConceptZoo/JavaScript/OPPConceptsZoo/utils/MongoConnection.js
import { MongoClient } from "mongodb";

const uri =
    "mongodb+srv://odalys:odalys@cluster0.2cf9puv.mongodb.net/?appName=Cluster0";

const client = new MongoClient(uri);

export async function connect() {

    await client.connect();

    return client.db("FarmDB");

}